src/components/icons/clockOutlinedTeal.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+const ClockOutlinedTeal = (props) => {
+  return (
+    <svg viewBox="0 0 22 22" {...props}>
+      <g
+        transform="translate(-5 -5)"
+        stroke="#3696B9"
+        strokeWidth={2}
+        fill="none"
+        fillRule="evenodd"
+      >
+        <rect x={6} y={6} width={20} height={20} rx={10} />
+        <path strokeLinecap="round" d="M16 11.2V16l2.4 2.4" />
+      </g>
+    </svg>
+  )
+}
+
+export default ClockOutlinedTeal
```

Comment:
oh! this should be a png and be converted to a component automatically instead of being placed manually here - or am I wrong? I can't fully recall how the setup is in components-pkg

Reply:
No, we don't have that set up in components yet. We do have a tech-20 about icons in components in general, I'd tackle them together

Reply:
Mmmmm good question, I will check it.
Thank you 🍦

Reply:
thank you for the clarification @Averethel 🧸